DogeNet.BLL/Features/Posts/EditPost/EditPostHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿// <copyright file="EditPostHandler.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Posts.EditPost
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using DogeNet.BLL.Features.Posts.GetPost;
+    using DogeNet.BLL.Services.Interfaces;
+    using DogeNet.DAL;
+    using DogeNet.DAL.Models;
+    using MediatR;
+
+    public class EditPostHandler : IRequestHandler<EditPostCommand, PostDetailsModel>
+    {
+        private readonly DBContext context;
+
+        private readonly IMapper mapper;
+
+        public EditPostHandler(DBContext context,  IMapper mapper)
+        {
+            this.context = context;
+            this.mapper = mapper;
+        }
+
+        public async Task<PostDetailsModel> Handle(EditPostCommand request, CancellationToken cancellationToken)
+        {
+            var post = this.context.Posts.Find(request.model.Id);
+
+            post.Text = request.model.Text;
```

Comment:
Could we use automapper here?

Reply:
ok, working on it =D

Reply:
@oleh-leobit  actually I don't understand how to change one object in memory with another one
so, I can map model into db object, but how can I change existing obj in context with mapped obj?
![image](https://user-images.githubusercontent.com/77879337/160837284-e53af5aa-e743-4a42-a936-ee3f2e82aff8.png)

Reply:
https://stackoverflow.com/questions/2374689/automapper-update-property-values-without-creating-a-new-object

Reply:
thanks, exactly what I need